content/en/intelligent_test_runner/setup/java.md

```diff
@@ -22,13 +22,14 @@ further_reading:
 
 ## Compatibility
 
-Intelligent Test Runner is supported in `dd-java-agent >= 1.22.0`.
+Intelligent Test Runner is supported in `dd-java-agent >= 1.26.0`.
```

Comment:
I keep bumping the version here since Java ITR is still in beta, and fixes keep getting added as issues are discovered. Earlier versions of the tracer support ITR too, but they have known bugs - which is why I want the clients to use the newest version.